spring-integration-redis/src/main/java/org/springframework/integration/redis/inbound/RedisQueueMessageDrivenEndpoint.java

```diff
@@ -80,6 +80,8 @@
 
 	private volatile Runnable stopCallback;
 
+	private volatile boolean readFromLeft = false;
```

Comment:
If the option is named `tail` in both inbound and outbound adapters, it might be counter intuitive to some users that the default is `true` on inbound, and `false` in outbound (assuming that current behavior is retained).